website/docs/pipelines/promoting-applications.mdx

```diff
@@ -154,21 +168,26 @@ promotion:
 
 More info in the [spec](../spec/v1alpha1/pipeline/#pipeline)
 
-### Create credentials secret
+### Credentials Secret
 
-In order to authenticate with the Git provider repository hosting the manifests, you need to create a Secret with credentials used for cloning.
-The contents of this Secret are the same you use for [GitRepositories](https://fluxcd.io/flux/components/source/gitrepositories/#secret-reference).
+In the journey of creating a pull request, there are different secrets involved:
 
-The pipelines controller will be creating pull requests against this repository so you will need an API token in addition to the `git` credentials. You may want to create a bot account with clone and pull request permissions for this operation.
+1. Pipeline controller receives events via [webhook from leaf clusters](./#setup-notifications-from-leaf-clusters). Hmac is used for authN/authZ so a hmac key should be provided in this case.
+2. Pipeline controller clones and patches manifests to promote from the pipeline configuration repo via https or ssh. A set of [git credentials](https://fluxcd.io/flux/components/source/gitrepositories/#secret-reference) are required.
```

Comment:
```suggestion
2. Pipeline controller clones and patches manifests to promote from the pipeline configuration repo via https. A set of [git credentials](https://fluxcd.io/flux/components/source/gitrepositories/#secret-reference) are required.
```
I don't think we support ssh for cloning yet 😕

Reply:
Looking at the code, i think we do? https://github.com/weaveworks/pipeline-controller/blob/main/server/strategy/pullrequest/clone.go#L14

Reply:
changed